recipes/libconfig/all/conanfile.py

```diff
@@ -1,81 +1,98 @@
+from conan import ConanFile
+from conan.tools.apple import is_apple_os
+from conan.tools.files import get, copy, rmdir, replace_in_file
+from conan.tools.scm import Version
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+
 import os
-from conans import ConanFile, CMake, tools
 
+required_conan_version = ">=1.53.0"
 
 class LibconfigConan(ConanFile):
     name = "libconfig"
+    description = "C/C++ library for processing configuration files"
     license = "LGPL-2.1"
```

Comment:
Can you update this to `LGPL-2.1-only` or `LGPL-2.1-or-later`?

Reply:
Thank you for your review.
I think `LGPL-2.1-or-later` is appropriate. https://github.com/hyperrealm/libconfig/blob/master/lib/libconfig.hh#L9-L10